.github/CODEOWNERS

```diff
@@ -137,4 +137,5 @@ ui/components/ui/deprecated-networks                  @MetaMask/metamask-assets
 ui/components/ui/nft-collection-image                 @MetaMask/metamask-assets
 
 # Extension Platform
-yarnrc.yml                                            @MetaMask/extension-platform
\ No newline at end of file
+yarnrc.yml                                            @MetaMask/extension-platform
+mock-e2e-whitelisted.js                               @MetaMask/extension-platform
```

Comment:
ℹ️ I've added this into the platform team codeowners, for our team to ensure that no new urls/hosts are added in the whitelisted file (only removed)